source/source.json

```diff
@@ -6110,11 +6112,299 @@
-	}
+		"script_message": "You will need to have \"Game Patching\" and \"Loading external FIRMs and modules\"\nenabled in LumaCFW settings (hold select on boot)"
+	},
+	{
+		"github": "awkitsune/IkuReader",
+		"title": "IkuReader",
+		"systems": ["DS"],
+		"categories": ["utility"],
+		"description": "An ebook reader",
+		"long_description": "Ebook reader compatible with fb2, epub and txt file formats. Updated original source code to be compatible with DSi, TMFH and HiyaCfw.",
+		"icon": "https://raw.githubusercontent.com/awkitsune/IkuReader/main/data/icon.bmp"
+	},
+	{
+		"github": "TehFridge/ShutTheCamUp",
+		"title": "Shut The Cam Up",
+		"systems": ["3DS"],
+		"categories": ["utility"],
+		"description": "Removes the camera shutter and high-pitched beep sound from the Nintendo 3DS (works on all 3ds apps that use the camera)",
+		"long_description": "# DISCLAIMER\nI do not condone any misuses of this patch for malicious purposes or whatever.\n\n# Why tho?\nImagine you are playing your 3ds in a cafe or smt, you need to use the camera feature in some game (for example. Face Raiders) and suddenly the shutter sound bursts out of the speakers and people look at you weirdly, we wouldnt want that. \n\nAlso like... phones don't do that sort of thing (well maybe in japan, china cause there the law demands that anything with a camera has to have a shutter sound)\n\nI mostly created this patch cause i like the vibe of vlogging with a 3ds. It has its charm you know.\n\n# How is it possible?!\nThe shutter sound was embedded in some system process, we just discovered it now lol. So i just decided to null the bytes with audio data, and it worked!\n\n# Does it work on both Old and New 3DS?\nYup. there are two seperate patches individually for the new and old model of the console.",
+		"icon": "https://cdn.discordapp.com/attachments/1146918310072549386/1146918328389087312/ikonka2.png",
+		"scripts": {
+			"Install the patch (Old 3DS/2DS)": [
+				{
+    					"type": "promptMessage",
+    					"message": "Ready to install the patch ?\n(Old 3DS/2DS)"
+				},
+				{
+					"type": "mkdir",
+					"directory": "sdmc:/luma/titles/"
+				},
+				{
+					"type": "mkdir",
+					"directory": "sdmc:/luma/titles/0004013000001602/"
+				},
+				{
+    					"type": "downloadRelease",
+    					"repo": "TehFridge/ShutTheCamUp",
+    					"file": "old3ds.ips",
+    					"output": "sdmc:/luma/titles/0004013000001602/code.ips",
+    					"includePrereleases": false
+				},
+				{
+					"type": "mkdir",
+					"directory": "sdmc:/luma/sysmodules/"
+				},
+				{
+    					"type": "downloadRelease",
+    					"repo": "TehFridge/ShutTheCamUp",
+    					"file": "old3ds.ips",
+    					"output": "sdmc:/luma/sysmodules/0004013000001602.ips",
+    					"includePrereleases": false
+				}
+			],
+			"Install the patch (New 3DS/2DS)": [
+				{
+    					"type": "promptMessage",
```

Comment:
unneeded

Reply:
the prompt message isn't needed, but the separate patch installation is apparently. There are details on the GitHub that the two patches are different, in this script it downloads for the new consoles

Reply:
yeah, just a not needed prompt